Bottom: de5030388d3a0f9325c63bd934f6ab0fc13903e4
Top:    b640fcf4f0f69f8cb52a36ad028ac0109aeae658
Author: Robert Berger <robert.berger@ReliableEmbeddedSystems.com>
Date:   2021-01-07 15:57:26 +0100

hack poky.conf to get rid of snapshot

Signed-off-by: Robert Berger <robert.berger@ReliableEmbeddedSystems.com>


---

diff --git a/meta-poky/conf/distro/poky.conf b/meta-poky/conf/distro/poky.conf
index 623f39da66..4d909e3ed5 100644
--- a/meta-poky/conf/distro/poky.conf
+++ b/meta-poky/conf/distro/poky.conf
@@ -1,7 +1,9 @@
 DISTRO = "poky"
 DISTRO_NAME = "Poky (Yocto Project Reference Distro)"
-DISTRO_VERSION = "3.2+snapshot-${METADATA_REVISION}"
-DISTRO_CODENAME = "master"
+#DISTRO_VERSION = "3.2+snapshot-${METADATA_REVISION}"
+DISTRO_VERSION = "3.2.66"
+#DISTRO_CODENAME = "master"
+DISTRO_CODENAME = "master-as-gatesgarth"
 SDK_VENDOR = "-pokysdk"
 SDK_VERSION = "${@d.getVar('DISTRO_VERSION').replace('snapshot-${METADATA_REVISION}', 'snapshot')}"
 SDK_VERSION[vardepvalue] = "${SDK_VERSION}"
